### PR TITLE
implemented py-copy-file

### PR DESCRIPTION
**Pull request by mariiahorbova**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    source_file = parts[1]
+    destination_file = parts[2]
```
Comment (Andriy-Sydorenko) [inline app/main.py:7]: It's better to use `try` for the whole function, to catch some `ValueError` exceptions, e.g. if the user is trying to copy the file to itself. or if the input is invalid or isn't splittable to 3 elements (f.e. "cp smth_smth"), you need to raise an appropriate exception, then catch them in `except`, and print the error message.

**Review by Andriy-Sydorenko — CHANGES_REQUESTED** [review]

**Review by Andriy-Sydorenko — APPROVED** [review]
Good job!))

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]
_From checklist:_
Make sure you check that the `cp` input command is correct.

**Comment by Oleg-s-s** [comment]
You've already implemented this check, but deleted it with len() check

**Review by Oleg-s-s — APPROVED** [review]
:+1:

**Pull request closed without merging**